### DISCUSSION TITLE
existing codespace is not accessible around 00:00 UTC 19/JUL/2024.

### DISCUSSION THREAD

**Discussion by Leqiangw** [community/community#132736] [Codespaces]
### Select Topic Area

General

### Body

I failed to access one existing codespace around 00:00 UTC today. 
The error I got like:
<img width="396" alt="codespaceerror" src="https://github.com/user-attachments/assets/43d40db6-b390-45ce-8efe-de4b29a47ea4">
![Screenshot 2024-07-19 141213](https://github.com/user-attachments/assets/66b97f8d-086e-4a26-baab-1166827a87e8)

The things I tried:
1. reload the page, tried several times.
2. restart codespace, tried several times
3. connect codespace with vscode and also web browser.

vscode version is 1.91.1
Has anyone got this issue before? Thanks.

Regards,
Lucas

**Comment by Leqiangw** [comment]
anyway, it's finally recovered after 2.5hours.

**Reply by dorhaLuo** [reply]
how did you solve it? I'm getting the exact same error message

**Reply by dorhaLuo** [reply]
oh I see what you mean, it recovered by itself :D
